src/Faker/Core/Color.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Faker\Core;
+
+use Faker\Extension;
+
+/**
+ * @experimental This class is experimental and does not fall under our BC promise
+ */
+final class Color implements Extension\ColorExtension
+{
+    protected $allColorNames = [
```

Comment:
Should we maybe mark these as CSS color names instead? Else there will be confusion for localized ones in the future. Same goes for the safe ones